pkg/pb/bbgo.proto

```diff
@@ -150,8 +150,9 @@ message CancelOrderResponse {
 }
 
 message QueryOrderRequest {
-    int64 id = 1;
```

Comment:
would prefer using string here, because some exchange use UUID (string).

we can parse the integer from the server side.

Reply:
ok